x-pack/plugin/inference/src/yamlRestTest/resources/rest-api-spec/test/inference/70_text_similarity_rank_retriever.yml

```
@@ -0,0 +1,90 @@
+setup:
+  - requires:
+      cluster_features: "gte_v8.15.0"
```

```suggestion
      cluster_features: "text_similarity_reranker_retriever_supported"
```
You should be able to use your node feature as declared.